tests.sh

```diff
@@ -226,10 +226,11 @@ internal_install_python_deps() {
   if [ "$TRAVIS" != "true" ]; then
     return;
   fi
-  # Install tox (OS X doesn't have pip).
+  # Install 
+  (OS X doesn't have pip).
```

Comment:
Missing "#" add the beginning of the line

ps: You should squash (i.e. fixup) all your commits to have atomic change and all commit which compile (so you can fast forward on master instead of merging if you want...)

Reply:
Fixed. The tests are now passing, thanks for your help!

Will squash into on commit